pkg/metrics/metrics.scss

```diff
@@ -551,16 +554,26 @@ $graphs: cpu, memory, disks, network;
     }
 
     .polygon.polygon-use {
-      transform: scale(-1);
+      transform: scaleX(-1);
```

Comment:
This seems like the most suspicious candidate for the metrics regression. `scaleX` is [not documented](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform), but I assume it is essentially a mirror/flip along the Y axis (i. e. in horizontal, X direction), instead of scaling both coordinates with -1 (which flips vertically as well). This actually makes more sense, but I figure the polygon computation also goes the wrong way around in Y, so we need to fix that there. I'll look into that tomorrow.

Reply:
I added another commit which fixes the graphs. Previously, the sat graphs were the wrong way around, this PR broke the use graphs. Now it should be good in all cases (use/sat/LTR/RTL). I trigger another pixel run.

Reply:
Now it looks correct both in [LTR](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18774-20230510-200557-cea87f2e-fedora-37/pixeldiff.html#TestHistoryMetrics-testEvents-metrics-history-expanded-hour) and [RTL](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18774-20230510-200557-cea87f2e-fedora-37/pixeldiff.html#TestHistoryMetrics-testEvents-metrics-history-expanded-hour-rtl)